datasets.py

```diff
@@ -408,6 +408,11 @@ def __init__(self, transform):
         self.categories = list(self.labels_to_names.keys())
         self.attribute_names = ["Female", "Male"]
         self.scene_mapping = NoneDict()
+
+        # If attribute is ordinal, set self.ordinal = True and define self.axis = [integers with indexed order of attributes]
```

Comment:
maybe give an example of what it means for an attribute to be ordinal here? can give example of skin tone, age, etc.